File: functions/function_notes.c
//Adelle Ponczoch, Function Notes in C
#include <stdio.h>

char name;

int add(void){
    return 6+8;
}

void add(int numOne,int numTwo){
    printf("%d\n", numOne+numTwo);
}
const char* input(char type[50], int length){
    char answer[50];
    printf("Please give me a %s:\n", type);
    //getStr(answer, sizeof(answer)-1);
    return answer;
}
int main(void){
    //printf("Hello World\n");
    //add(4,9);
    //add(1000,2);
    //add(47,9);
    //printf("%d", 6+8);
    //char name = word("name");
    //char verb = word("verb");
    //char place = word("place");
    //printf("%s was %s until they somehow reached %s", word("name", 50), word("verb", 50), word("place", 50));
    return 0;
}

File: functions/function_notes.py
# Adelle Ponczoch, Function Notes in Python

#function is an action stored in a keyword

#def add():
   # numOne = int(input("Give me a number:\n"))
   # numTwo = int(input("Give me a number:\n"))
   # print(numOne + numTwo)

#add()
#add()

#def add(numOne, numTwo): #parameters go in the parenthesis separated by commas
    
  #  print(numOne+numTwo)

#add(12,20)#arguments are given when the function is called AND must match the number of parameters
#add(2,4)
#add(7,21)
#number = int(input("Can I get a number:\n"))
#def add(numOne, numTwo): #parameters go in the parenthesis separated by commas
    
   # print(numOne+numTwo)

#add(int(input("Tell me a number:\n")), number)#arguments are given when the function is called AND must match the number of parameters
#add(2,4)
#add(7,21)

def user(word):
    return input(f"Tell me a {word}:\n")

name = user("name")
verb = user("verb")
place = user("place")
print(f"{name} was {verb} and somehow got to {place}")